ert_shared/main.py

```diff
@@ -458,5 +458,19 @@ def main():
         sys.exit(str(err))
     except:
         logger.exception("ert crashed unexpectedly")
-        sys.exit("ert crashed unexpectedly")
+
+        logfiles = set()  # Use set to avoid duplicates...
+        all_loggers = [  # Call getLogger() to ensure the logger is initialised
+            logging.getLogger(name) for name in logging.root.manager.loggerDict
+        ]
+        # List-comprehension above does not include root-logger, add it manually
+        for lgger in [logging.getLogger()] + all_loggers:
+            for handler in lgger.handlers:
+                if isinstance(handler, logging.FileHandler):
+                    logfiles.add(handler.baseFilename)
```

Comment:
Might be a bit exessive as this gives us all the loggers that exist for that run. The root logger should be sufficient, as it logs everything.

Reply:
It lists all unique files found in all `FileHandlers`. In my setup running the poly-example, I get this
```
MacBook-Air:poly_example bjarne$  ert ensemble_experiment poly.ert
ERT crashed unexpectedly. See logfiles for details:
   /Users/bjarne/git/Equinor/ert/test-data/local/poly_example/ee-log.txt
   /Users/bjarne/git/Equinor/ert/test-data/local/poly_example/api-log.txt
   /Users/bjarne/git/Equinor/ert/test-data/local/poly_example/ert-log.txt
MacBook-Air:poly_example bjarne$ 
```
The rationale is that unexpected exceptions may happen anywhere (aka _unexpected_ 😁 ), and we actually don't really know how logging is set up, so relevant information may appear in any of these files AFAICS.

But if you still feel this is too exessive I'm happy to produce less code 😄

Reply:
I do feel it is a bit excessive, I think you addition of the error message and the path to the root logger is good though.